printModules/printHeader.js

```diff
@@ -1,13 +1,14 @@
 //Switch
 import validateChain from "../chainModules/validateChain.js";
 import printPage from "./printPage.js";
+import loginBtn from "../userModules/login/loginBtn.js";
 
 export default function printHeader(state) {
-    let currentUser = "";
-    if (state !== "logInSuccess") {
+  let currentUser = "";
+  if (state !== "logInSuccess") {
 
-        headerDiv.innerHTML = `<div id="validationDiv"><div id="validationResultDiv"></div><button id="validationButton">Validate</button></div>
-      <form id="logInForm">
```

Comment:
varför tar vi bort form? då kan man inte klicka enter längre för att logga in

Reply:
Form laddar om sidan när man klickar på knappen, och vi använder eventListener som sedan hämtar värdena ur fälten. Form output används inte i föregående kod, koden hämtar och kollar bara value av fälten.

Reply:
form med (e)
e.preventDefault 
:)